### PR TITLE
Remove advertisements from course search page

### LINKED ISSUE #238: Remove Advertisements
Remove advertisements from the class search page.

### PR DESCRIPTION
**Pull request by kartik-bhagatwala**
## Summary
Remove the advertisements from course search page. Backend routes for ads are unchanged. 

## Test Plan
Test to see if ads and ad disclaimer are gone.
Before:
Screen Shot 2022-02-12 at 3 52 19 PM

After:
Screen Shot 2022-02-12 at 3 51 48 PM

## Issues
Resolves #239 

## Future Followup (Optional)
Maybe delete the AdBanner component? Kept for now in case we want to re-enable them in the future.

### CODE REVIEW

File: client/src/components/CoursePane/CourseRenderPane.js [hunk 1]
```diff
@@ -7,7 +7,6 @@ import darkNoNothing from './static/dark-no_results.png';
 import RightPaneStore from '../../stores/RightPaneStore';
 import loadingGif from '../SearchForm/Gifs/loading.gif';
 import darkModeLoadingGif from '../SearchForm/Gifs/dark-loading.gif';
-import AdBanner from '../AdBanner/AdBanner';
 import { RANDOM_AD_ENDPOINT } from '../../api/endpoints';
```
Comment (ChaseC99) [inline client/src/components/CoursePane/CourseRenderPane.js]:
```suggestion

```
Unused import

**Review by ChaseC99 — CHANGES_REQUESTED** [review]
Dang so quick getting this pr out!

Everything looks solid. Let's just also delete the `AdBanner` component now. We don't want dead code in the project. We can always get it from the git history if we decide to reimplement.

**Comment by kartik-bhagatwala** [comment]
Thanks for the input Chase, I've made those changes you suggested.

**Review by ChaseC99 — APPROVED** [review]
LGMT! Great work 🚢

**Pull request merged**